data/json/items/gunmod/bore.json

```diff
@@ -33,8 +33,8 @@
     "symbol": ":",
     "color": "green",
     "location": "bore",
-    "mod_targets": [ "rifle" ],
```

Comment:
It may be more realistic to create new conversion kits instead of extending these.